app/code/Magento/Multishipping/Test/Mftf/ActionGroup/StorefrontChangeMultishippingItemQtyActionGroup.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontChangeMultishippingItemQtyActionGroup">
+        <annotations>
+            <description>Change multishipping item quantity on Ship to Multiple Addresses page. Starts on Ship to Multiple Addresses page</description>
```

Comment:
I'm not sure what this part means: `Starts on Ship to Multiple Addresses page`. Can we remove it?
```suggestion
            <description>Change multishipping item quantity on Ship to Multiple Addresses page.</description>
```

Reply:
Removed